shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -96,7 +96,11 @@
   public static native boolean nativeGetIsSoftwareRenderingEnabled();
 
   @UiThread
```

Comment:
Should `@UiThread` still be applied to be this method?

Reply:
Removed the annotation